.github/workflows/common.yml

```diff
@@ -15,17 +15,11 @@ on:
       - "v*"
     branches:
       - main
-      - devel
-      - patch
```

Comment:
we actually need the patch one as we still don't want to release everything from `main` when doing a patch/bug fix

Reply:
After you release from patch, you'd merge to main and the site would be built then. So do we need the patch pkgdown builds?

Reply:
but i think we still want the workflows to trigger when PRing into `patch`?  IF we remove `patch` then it won't trigger any of the checks - I thought??